app/packs/src/apps/mydb/elements/list/ElementsTable.js

```diff
@@ -269,6 +295,60 @@ export default class ElementsTable extends React.Component {
     );
   };
 
+  attachmentTokenModal = () => {
+    const { showAttachmentTokenModal, attachmentTokens, showEmpty } = this.state;
```

Comment:
It looks like showEmpty is declared but unused. Might be better to remove it.

Reply:
Done